turbopack/crates/turbopack/src/module_options/mod.rs

```diff
@@ -486,8 +506,21 @@ impl ModuleOptions {
 
                 rules.push(ModuleRule::new(
                     RuleCondition::Any(vec![
-                        RuleCondition::ResourcePathEndsWith(".css".to_string()),
-                        RuleCondition::ContentTypeStartsWith("text/css".to_string()),
+                        RuleCondition::All(vec![
+                            RuleCondition::Any(vec![
+                                RuleCondition::ResourcePathEndsWith(".css".to_string()),
+                                RuleCondition::ContentTypeStartsWith("text/css".to_string()),
+                            ]),
+                            RuleCondition::not(module_css_condition.clone()),
+                        ]),
+                        RuleCondition::All(
+                            [
+                                vec![module_css_condition.clone()],
+                                // see comment on module_css_external_transform_conditions
+                                module_css_external_transform_conditions.clone(),
```

Comment:
This list of conditions need to be in a RuleCondition::Any.

So it need to be:

```
RuleCondition::All(vec![
    module_css_condition.clone(),
    RuleCondition::Any(module_css_external_transform_conditions.clone())
])
```